azavea/fb-gender-survey-dashboard
# Add Netlify rewrite directive

## Overview

When deployed, URLs should simply load the root index file and use
pushState to keep URLs intact. The application will then handle routing
locally.

Connects #43 

### Notes

Rather than add a redirect for any route to `/`, I added a redirect instead. This way, if we do add routes that we want to preserve (for example, the saved state route), the app can continue to serve it. In practice, the main pages each check for the existence of state they need to operate, and redirect to `/` internally. 

## Testing Instructions

 * In the deploy preview of this PR, navigate to routes within the app and refresh the page. Ensure that the app redirects and does not give a 404 error.
